source/locale/pt_BR/LC_MESSAGES/architecture.po

```diff
@@ -84,21 +84,20 @@ msgid ""
 "directly using the REST APIs provided by the API Gateway. Data processing "
 "flows might be also configured - these entities can perform a variety of "
 "actions, such as generate notifications when a particular device attribute "
-"reaches a certain threshold or save all data generated by a device onto an "
-"external database. As devices start sending their readings to dojot, a user "
+"reaches a certain threshold or send data generated by a device to an "
+"external service. As devices start sending their readings to dojot, a user "
 "can:"
 msgstr ""
 "Usando a dojot: um usuário configura dispositivos de IoT por meio da GUI ou "
 "diretamente usando as APIs REST fornecidas pelo API Gateway. Os fluxos de "
-"processamento de dados também podem ser configurados - essas entidades "
-"podem executar uma variedade de ações, como gerar notificações quando um "
-"atributo de dispositivo específico atingir um determinado limite ou salvar "
-"todos os dados gerados por um dispositivo em um banco de dados externo. À "
-"medida que os dispositivos começam a enviar suas leituras para dojot, um "
-"usuário pode:"
+"processamento de dados também podem ser configurados - essas entidades podem "
+"executar uma variedade de ações, como gerar notificações quando um atributo "
+"de dispositivo específico atingir um determinado limite ou salvar todos os "
+"dados gerados por um dispositivo em um banco de dados externo. À medida que "
+"os dispositivos começam a enviar suas leituras para dojot, um usuário pode:"
 
 #: ../../source/architecture.rst:41
-msgid "receive these readings in real time by socket.io channels;"
+msgid "receive these readings in real time by socket.io or websocket channels;"
 msgstr "receber essas leituras em tempo real pelos canais *socket.io*;"
```

Comment:
include `websocket` in Portuguese translation

Reply:
Included websocket in Portuguese translation